shared/dev/dumb-sheet/render.native.js

```diff
@@ -262,7 +262,7 @@ class DumbSheetRender extends Component<void, Props, any> {
         <Box style={styleBox}>
           <Text type="BodySmall">{key}: {mockKey}</Text>
           <Box style={styleSmallScreen}>
-            <Box style={{flex: 1}} {...mock.parentProps}>
+            <Box {...mock.parentProps}>
```

Comment:
See the commit message here https://github.com/keybase/client/pull/7244/commits/f39df51a8ca3d7d4d951a26882ea95bb808ccd3d

It doesn't seem to break other dumb screens, but it does fix this issue I ran into.